config/intellij-idea-inspections.xml

```diff
@@ -3025,6 +3026,8 @@
                    enabled_by_default="false"/>
   <inspection_tool class="MagicNumber" enabled="true" level="ERROR" enabled_by_default="true">
     <scope name="Tests" level="ERROR" enabled="false"/>
+    <!-- Temporary method until https://github.com/checkstyle/checkstyle/issues/13426 -->
+    <scope class="ViolationMessagesMacro" level="ERROR" enabled="false"/>
```

Comment:
Please share what force you to do this

Reply:
Removed suppression. Same as https://github.com/checkstyle/checkstyle/pull/13423#discussion_r1280641475